index.html

```diff
@@ -56,6 +56,127 @@ <h1>
         </li>
     </ul>
 </div>
+<div class="readmore-button">
+    <a>
+        <img src="./images/icons/readmore-icon.svg" alt="read more ">
+    </a>
+</div>
+</section>
+<section id="works" >
+    <div class="recent-works">
+        <h2>My Recent </br> Works</h2>
+    </div>
+    <div class="myworks-card">
+        <div class="myworks-thumbnail">
+            <img  alt="">
+            <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+            <ul class="work-languages" >
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+            </ul>
+            <button type="button" > See Project</button>
+        </div>
+    </div>
+    <div class="myworks-card">
+        <div class="myworks-thumbnail">
+            <img  alt="">
+            <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+            <ul class="work-languages" >
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+            </ul>
+            <button type="button" > See Project</button>
+        </div> 
+    </div>
+    <div class="myworks-card">
+        <div class="myworks-thumbnail">
+            <img  alt="">
+            <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+            <ul class="work-languages" >
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+            </ul>
+            <button type="button" > See Project</button>
+        </div> 
+    </div>
+    <div class="myworks-card">
+        <div class="myworks-thumbnail">
+            <img  alt="">
+            <h2 class="work-title">Multi-Post Stories Gain+Glory</h2>
+            <ul class="work-languages" >
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                <li>HTML</li>
+            </ul>
+            <button type="button" > See Project</button>
+        </div> 
+    </div>
+</section>
+<section class="about-me">
+    <div class="container-aboutme">
+        <h2>About me</h2>
+        <div class="aboutme-details">
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+			see and have a project you need coded,
+			don’t hestiate to contact me.</p>
```

Comment:
Thank you for your good work on the about myself section. As per the Figma template, I have a suggestion that can improve the look of your paragraph:

- [ ] Kindly adjust the `line-height` of your paragraph to match the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-988&t=yCafCmU4xSO1HuYd-0), this will improve the readability of your content and make it look closer to the design.

  ![image](https://user-images.githubusercontent.com/107952724/235262521-896685d0-2e9f-43a1-91da-a75c78d7d975.png)